src/python/pants/backend/python/lint/pylint/rules.py

```diff
@@ -60,36 +64,64 @@ async def pylint_lint(
     python_setup: PythonSetup,
     subprocess_encoding_environment: SubprocessEncodingEnvironment,
 ) -> LintResult:
-    if pylint.options.skip:
+    if pylint.skip:
         return LintResult.noop()
 
     # Pylint needs direct dependencies in the chroot to ensure that imports are valid. However, it
     # doesn't lint those direct dependencies nor does it care about transitive dependencies.
```

Comment:
Given this, the renaming of this variable is maybe a bit misleading... we add the direct deps, but they're not linted.

Reply:
True. The rename was prework for https://github.com/pantsbuild/pants/pull/9792, where I want to distinguish from `plugin_targets`. I'm not sure what to call this variable. `targets` won't be descriptive anymore. `lint_targets_with_dependencies`? It's verbose, but true.